src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixExtension.groovy

```diff
@@ -7,7 +7,7 @@ import org.gradle.api.provider.SetProperty
 
 class ScalafixExtension {
 
-    private static final String DEFAULT_CONFIG_FILE = ".scalafix.conf"
+    protected static final String DEFAULT_CONFIG_FILE = ".scalafix.conf"
```

Comment:
i made this change before #9 was put in. if you like i can change this back to private and modify the tests to pass rules via command line instead of writing out a conf file

Reply:
it is ok to write the file. Some rules require additional configuration (e.g. `DisableSyntax`) which can only be provided via config file

Reply:
we have the `.scalafix.conf` string repeated in some tests already. It is not a big deal if you need to add another one.

Reply:
ok, cool. didnt realize that was repeated already. i will change it back to private